.github/workflows/check-links.yml

```diff
@@ -61,8 +61,6 @@ jobs:
           "repository/**/*.md"
           "repository/**/*.py"
           "documentation/dev/**/*.html"
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Note that I've removed these 2 lines since they aren't needed anymore with https://github.com/lycheeverse/lychee-action/pull/196